src/init/index.ts

```diff
@@ -1,13 +1,14 @@
 import * as Generator from "yeoman-generator";
 import * as chalk from "chalk";
 import { GlobalJson, FabloConfigJson } from "../types/FabloConfigJson";
+import { version } from "../../package.json";
 
 function getDefaultFabloConfig(): FabloConfigJson {
   return {
-    $schema: "https://github.com/hyperledger-labs/fablo/releases/download/2.2.0/schema.json",
+    $schema: `https://github.com/hyperledger-labs/fablo/releases/download/${version}/schema.json`,
     global: {
       fabricVersion: "2.5.12",
-      tls: false,
+      tls: true,
```

Comment:
It is wort adding to the changelog file - that starting from this PR the init command generates network with TLS/RAFT instead of non-TLS/solo